Provis.Core/Services/UserService.cs

```diff
@@ -97,5 +112,32 @@ public async Task<UserActiveInviteDTO> IsActiveInviteAsync(string userId)
 
             return userActiveInviteDTO;
         }
+
+        public async Task UpdateUserImageAsync(IFormFile img, string userId)
+        {
+            var user = await _userManager.FindByIdAsync(userId);
+
+            string newPath = await _fileService.AddFileAsync(img.OpenReadStream(), _imageSettings.Value.Path, img.FileName);
```

Comment:
Use async methods for work with streams

Reply:
There isn't async method for openReadStream